### PR TITLE
environment-map global scope

### PR DESCRIPTION
**Pull request by nedmsmith**
Added text requiring environment-map to be globally unique

### CODE REVIEW

**Comment by nedmsmith** [comment]
This PR addresses issue #176

**Comment by deeglaze** [comment]
"Globally unique" doesn't really match my understanding given that an environment can be elaborated to have an instance field to make it even-more-unique for matching instances against reference values for classes.

**Comment by deeglaze** [comment]
In the meeting you said just that the class-id needs to be globally unique, which makes more sense to me.

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -777,6 +777,9 @@ context (triple) in which the environment is used.
 An environment is named after a class, instance or group identifier (or a
 combination thereof).
 
+An environment MUST be globally unique.
+The combination of values within `environment-map` may be combined so that the combination is globally unique.
```
Comment (andrew-draper) [inline draft-ietf-rats-corim.md:781]:
```suggestion
The combination of values within `environment-map` must combine to form a globally unique identifier.
```

**Review by yogeshbdeshpande — APPROVED** [review]

File: draft-ietf-rats-corim.md [hunk 2]
```diff
@@ -777,6 +777,9 @@ context (triple) in which the environment is used.
 An environment is named after a class, instance or group identifier (or a
 combination thereof).
 
+An environment MUST be globally unique.
```
Comment (andrew-draper) [inline draft-ietf-rats-corim.md:780]: For example, an environment with just `class-id` may denote a specific organization's SKU (product name and specifications), the `instance` field is unique within the `class-id` namespace.

Comment (andrew-draper) [inline draft-ietf-rats-corim.md:780]:
```suggestion
An environment MUST be globally unique.
For example, an environment with just `class-id` may denote a specific organization's SKU (product name and specifications), the `instance` field is unique within the `class-id` namespace.
```

**Comment by nedmsmith** [comment]
Might I suggest that additional clarification in the form of examples be added by a different PR? It seems there is consensus on the simple change this PR originally proposed.

**Review by nedmsmith — COMMENTED** [review]
I resolved the threads related to exemplary text since we can add this later.

**Review by andrew-draper — APPROVED** [review]
I think the proposed changes attached to this revision make the spec better.
I suggest we approve these now, and to put additional changes into another PR.

**Pull request merged**